src/zepto.js

```diff
@@ -51,9 +51,9 @@ var Zepto = (function() {
 
   zepto.matches = function(element, selector) {
     if (!selector || !element || element.nodeType !== 1) return false
-    var matchesSelector = element.webkitMatchesSelector || element.mozMatchesSelector ||
+    var matches = element.matches || element.webkitMatchesSelector || element.mozMatchesSelector ||
```

Comment:
Sure, please remove oMatchesSelector.

When doing both changes, please include all this rationale that we discussed in the commit message.